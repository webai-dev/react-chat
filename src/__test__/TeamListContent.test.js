import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import TeamListContent from "../common/components/ChatWidget/ChatList/TeamList/TeamListContent";
import renderer from "react-test-renderer";

it("renders without crashing", async () => {
  const { getByTestId } = render(<TeamListContent />);
  const teamListContent = getByTestId("team-list-content");
  expect(teamListContent).toBeTruthy();
});

it("matches snapshot", async () => {
  const tree = renderer.create(<TeamListContent />).toJSON();
  expect(tree).toMatchSnapshot();
});
