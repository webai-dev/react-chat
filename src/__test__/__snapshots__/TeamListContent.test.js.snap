// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="makeStyles-root-13"
  data-testid="team-list-content"
>
  <span />
</div>
`;
